website/source/docs/connect/proxies/built-in.md

```diff
@@ -9,11 +9,10 @@ description: |-
 # Built-In Proxy Options
 
 Consul comes with a built-in L4 proxy for testing and development with Consul
-Connect.
+Connect. Currently this does not support [L7 Traffic Management](https://www.consul.io/docs/connect/l7-traffic-management.html) and it is recommneded
+to use [Envoy with Connect](https://www.consul.io/docs/connect/proxies/envoy.html) if L7 traffic management is needed.
```

Comment:
It might be better to have this as part of the callout below.

Reply:
Done